cms/djangoapps/contentstore/views/utilities/tasks.py

```diff
@@ -0,0 +1,97 @@
+from celery.task import task
+from django.conf import settings
+from django.core.mail import send_mail
+from django.core.urlresolvers import reverse
+from smtplib import SMTPException
+
+from edxmako.shortcuts import render_to_string
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.inheritance import own_metadata
+
+from opaque_keys.edx.keys import CourseKey
+from xblock.fields import Scope
+
+
+def _get_course_problems(course_key):
+    """
+    Retrieve all problems in the course
+    """
+    # Note: Some components may have been orphaned due to a bug in split
```

Comment:
This could probably be folded into the docstring above.

Reply:
done